gqlalchemy/query_builder.py

```diff
@@ -579,7 +592,21 @@ def to(
             type_str = to_cypher_labels(relationship._type)
             properties_str = to_cypher_properties(relationship._properties)
 
-        self._query.append(EdgePartialQuery(variable, type_str, properties_str, bool(directed), False))
+        if algorithm is not None:
+            algorithm_str = str(algorithm)
+        else:
+            algorithm_str = ""
+
+        self._query.append(
+            EdgePartialQuery(
+                variable=variable,
+                labels=type_str,
+                algorithm=algorithm_str,
```

Comment:
`algorithm = "" if algorithm is None else str(algorithm)`

With this line You can remove if-else above return